lib/site-utils.js

```diff
@@ -33,16 +33,19 @@ exports.categories = Object.freeze({
 exports.findContentOnPage = function findContentOnPage($, selectors) {
   logger.log('selectors: %j', selectors);
 
+  // use the find() API in case we're handed an inner jquery object
+  const jQuery = _.isFunction($.find) ? $.find.bind($) : $;
```

Comment:
I couldn't just use `$.find` as the `jQuery` object, but rather bind it to the original `$` to get the function call to work.